docs/content/v1/usage/custom-directive.md

```diff
@@ -19,14 +21,76 @@ A `:` custom directive will interpret in JS whatever you pass as value.
 </script>
 ```
 
-outputs
+Outputs:
 
 ```html
 <a href="/user/144">Visit Profile</a>
 ```
 
-[Boolean attributes](https://www.w3.org/TR/html5/infrastructure.html#sec-boolean-attributes)
+## A Utility for Readability

```

Comment:
I have added this section because I use custom directives a lot for this purpose. I looked for this a while, because I used to use a lot of [computed properties in Vue](https://vuejs.org/guide/essentials/computed) and [derived states in Svelte](https://svelte.dev/tutorial/svelte/derived-state). It also illustrates how flexible LEGO is by using native JS.

Reply:
@vinyll What about this one?